updater/reports/ReportReposPersonalNonOwnerPushes.py

```diff
@@ -20,7 +20,8 @@ def query(self):
 		fourWeeksAgo = self.daysAgo(28)
 		query = '''
 			SELECT
-				CONCAT(users.login, "/", repositories.name) as "repository"
+				CONCAT(users.login, "/", repositories.name) as "repository",
+				COUNT(DISTINCT(pushes.pusher_id)) as "nonowner pusher"
```

Comment:
Could we make this plural (that is, `nonowner pushers`)?

Reply:
`nonowner pusher(s)`? 😄

Reply:
👍

Reply:
Fixed!

Reply:
Awesome, thanks!